compiler/fir/analysis-tests/testData/resolve/extendedCheckers/CanBeValChecker.kt

```diff
@@ -153,7 +153,7 @@ fun lambda() {
 }
 
 fun lambdaInitialization() {
-    <!CAN_BE_VAL!>var<!> <!VARIABLE_NEVER_READ!>a<!>: Int
+    var <!VARIABLE_NEVER_READ!>a<!>: Int
```

Comment:
`CAN_BE_VAL` is a warning, extended checker, so maybe minor, but `run` has:
```kt
    contract {
        callsInPlace(block, InvocationKind.EXACTLY_ONCE)
    }
```
so I think `a` is not captured, assigned exactly once, in place, hence can be `val`?

Reply:
Well, this `run` is not the one defined in standard library. It's defined above at line 144 as a contractless function.

Reply:
Aha, didn't look at its definition, sorry.

Reply:
That was a little confusing to me too, so maybe we can rename `run` to something like `exec` (same as compiler/testData/diagnostics/tests/controlFlowAnalysis/initializationInLambda.fir.kt)

Reply:
Usually we add prefix `my` to functions which shadows functions from stdlib: `run` -> `myRun`